tests/test_app.py

```diff
@@ -1,8 +1,95 @@
-from tests.utils import call_from_slack
+import random
+
+import pytest
+from tests.utils import call_from_slack, get_raw_block_text, respond_interactively
 
 
 def test_help_command(chalice_app):
-    r = call_from_slack(chalice_app, ["help"], "1234", "mattias")
+    r = call_from_slack(chalice_app, "help", f"{random.randint(0, 10000)}", "mattias")
```

Comment:
I would like to use the keyword arg style to make it more clear on what the argument is used for.
```suggestion
    r = call_from_slack(chalice_app=chalice_app, full_command="help", user_id=f"{random.randint(0, 10000)}", user_name="mattias")
```

There are more of these to edit if we agree on this.